README.md

```diff
@@ -269,8 +269,23 @@ Updates: `UPDATE Rewaya SET data_latest_version = 1 WHERE rewayaId = 3`.
   and run gradle custom build to generate the required APKs.
 
 #### 8.3 Windows
-
-TODO
+* Add Recitation Steps:
```

Comment:
How to know what rewayat already exist to select either Add or Update?
I would assume all rewayat from `mohafez-ios/.../recitations.csv` already exist with version 0, so I will always do the Update section. Is this correct?